client/src/components/ui/contextdrawer/InviteContext.svelte

```diff
@@ -14,10 +15,13 @@
 </script>
 
 <ContextTemplate bind:show={show}>
+    <p>You are removing the invitation for <b>{payload.email}.</b></p>
+    <ContextDivider />
```

Comment:
```suggestion
```
I recommend creating a confirmation modal instead of integrating the confirmation to the context menu.

Reply:
Did something cleaner

![image](https://user-images.githubusercontent.com/68340609/235836325-416e6b94-7337-4eb7-ab97-69a9c9842700.png)